src/main/java/org/terasology/climateConditions/ThermiaLevelChangedEvent.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.Event;
+
+public class ThermiaLevelChangedEvent implements Event {
```

Comment:
Please add some JavaDoc what this event is about? Consider making it `abstract` to indicate that a more specific event should be fired while it is okay to react upon the general event.

I'm wondering whether we have some kind of `BeforeAfterEvent<T>` already? This pattern of old and new value cannot be so uncommon... 🤔

Reply:
Replied here: https://github.com/Terasology/ClimateConditions/pull/17#discussion_r456828894
For now: changing name to AbstractThermiaLevelChangedEvent